### ISSUE TITLE
Haiti new operator

### ISSUE THREAD

**Issue by Francogo** [googlei18n/libphonenumber#746]
The following numbering plan for Haiti is not validated by the library.
The number are 995XXXXX, 996XXXXX, 997XXXXX, 998XXXXX, 999XXXXX, 980XXXXX, 981XXXXX, 982XXXXX, 983XXXXX and 985XXXXX.
If you need documentation about these number I can provide them to a private email address.
   
François

**Comment by roes** [comment]
Please send me an email at roes@google.com with the supporting documentation

**Comment by keghani** [comment]
Please send me an email at <my_username>@google.com with documentation for 984XXXXX.
(Replace <my_username> with keghani.)

**Comment by Francogo** [comment]
Sorry I don't have documentation about the 984XXXXXX. I have edited my issue to remove this prefix.

**Comment by dlyongemallo** [comment]
This will be fixed in the next version.

**Comment by Francogo** [comment]
I had a look to the version available today on https://libphonenumber.appspot.com/. I saw the problem is partially solved (numbers are accepted but carrier is not always correct). 
I know the issue is not yet closed but I would like to be sure it will be solved correctly.
995XXXXX, 996XXXXX, 997XXXXX, 998XXXXX and 999XXXXX are used by the carrier Access Haiti. 980XXXXX, 981XXXXX, 982XXXXX, 983XXXXX and 985XXXXX are used by Hainet.

The version available at https://libphonenumber.appspot.com/ is this the last stable version or the nightly build? It could be helpfull to add an indication with the version number on that page.

**Comment by keghani** [comment]
Thanks @Francogo!

The issue description here did not mention carriers, and as we're not fluent in French we did not read through the entirety of documents you'd sent. Sorry the carriers slipped.

I filed a separate issue at https://github.com/googlei18n/libphonenumber/issues/809 for the carriers.

The demo versoin is at https://github.com/googlei18n/libphonenumber/blob/master/README.md
It's not a nightly build, the version number corresponds to the release at https://github.com/googlei18n/libphonenumber/releases
